openedx_events/data.py

```diff
@@ -83,3 +85,38 @@ class EventsMetadata:
         ),
         validator=attr.validators.instance_of(tuple),
     )
+
+    def as_json(self):
```

Comment:
Pedantic complaint: JSON is a serialized data format, and so is always a string; we don't really have a good word for "data in a form that will readily serialize to JSON without further processing". A lot of code ends up just calling both "JSON", which I find confusing.

Here's a renaming suggestion:

- as_json -> as_json_data
- as_json_string -> to_json
- from_json_string -> from_json

What do you think?